cli.js

```diff
@@ -56,10 +74,44 @@ function run(file) {
 			return;
 		}
 
```

Comment:
Instead of running it directly here, how about we just return the file, and use `[].map` here: https://github.com/vdemedes/ava/blob/2651df6749af48ee985bad9b92d107d4770ea0eb/cli.js#L125 We can then run everything from here https://github.com/vdemedes/ava/blob/2651df6749af48ee985bad9b92d107d4770ea0eb/cli.js#L128 instead and use `Promise.all` for the execution instead of a global `files` counter.